spec/features/issues_spec.rb

```diff
@@ -292,6 +292,9 @@
         let(:trackable) { @issue }
         it_behaves_like 'a page with an activity feed'
 
+        let(:subscribable) { @issue }
+        it_behaves_like 'a page sith subscribe/unsunscribe links'
```

Comment:
Typo here. (same with the shared examples)

![palpatine](https://user-images.githubusercontent.com/1908272/42312266-7dfd6a10-8072-11e8-838d-5068c00ec7b9.jpg)

Reply:
😆